test_accounts/appeals_apis/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,45 @@
+
+## Authentication and Authorization
+
+The authentication model for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/appeals-status/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appeals-status/client-credentials)
+
+### Logging in with ID.me
+
+To log in to the sandbox environment using ID.me, make sure you:
+
+1. Choose an account that is specific to the scenario you want to test.
+2. Select **ID.me** to sign in to the sandbox environment.
+3. Enter the ID.me username and password. The password for all ID.me test accounts is: **Password1234!**
+4. **Don’t change any preselected answers** when asked about receiving an authentication code. Just click “Continue” to go to the next step.
+
+### Logging in with Login.gov
+
+To log in to the sandbox environment using Login.gov, make sure you:
+
+1. Choose an account that is specific to the scenario you want to test.
+2. Select **Login.gov** to sign in to the sandbox environment.
+3. Enter the Login.gov username and password. The password for all Login.gov test accounts is: **Password12345!!!**
+4. Use the Login.gov MFA seed to generate a 2FA code with an app such as Google Authenticator or Authy.
+
+## Veteran Test Accounts
+
+These accounts can be used to test various data and response scenarios for the `GET /appeals` endpoint. 
+
+`icn` and `X-VA-User` headers are required when making a request. `icn` should match the value in the ICN column, and `X-VA-User` should match the ID.me Username or Login.gov Username (depending on the authentication method used).
+
+| First Name | Last Name | Sandbox Response                                              | ICN               | ID.me Username                 | Login.gov Username        | Login.gov MFA Seed                                                                                                                                                                                              |
+|:-----------|:----------|:--------------------------------------------------------------|:------------------|:-------------------------------|:--------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Wesley     | Ford      | 1 Higher Level Review, 1 Supplemental Claim, 2 Legacy Appeals | 1012832025V743496 | va.api.user+idme.025@gmail.com | va.api.user+025@gmail.com | [46OPJSEX7URKZVICEKCARJVV7YRQRLFK](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B025%40gmail.com%29?secret=46OPJSEX7URKZVICEKCARJVV7YRQRLFK) |
```

Comment:
For better readability, consider putting the number of decision reviews in parentheses after the type [e.g. Higher-Level Review (1)].

Reply:
Thanks for the suggestion! I've made this change.